CountrySelect/country-options.tsx

```diff
@@ -0,0 +1,9 @@
+import * as R from 'ramda'
```

Comment:
I think we should avoid this transformation and simply keep the data in `country-options.tsx` in the same way as in `language-options.tsx`. It should be more consistent.